Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -44,15 +44,17 @@ class WebViewController: UIViewController, WebController {
     private var browserView = WKWebView()
     var onePasswordExtensionItem: NSExtensionItem!
     private var progressObserver: NSKeyValueObservation?
-    fileprivate var trackingProtecitonStatus = TrackingProtectionStatus.on(TrackingInformation()) {
+    private var trackingProtectionStatus = TrackingProtectionStatus.on(TPPageStats()) {
```

Comment:
TrackingInformation is now called TPPageStats. This was another change that was part of my TP PR that was committed locally and not pushed. In future, we need CI running on my commits, I just assumed they were without checking if that was the case.